libcontainer/cgroups/fscommon/rdma.go

```diff
@@ -0,0 +1,123 @@
+// +build linux
```

Comment:
We do not need build tags if we use filenames like `rdma_linux.go`

Reply:
resolved in latest commit.